snapd-glib/requests/snapd-json.c

```diff
@@ -763,6 +763,14 @@ _snapd_json_parse_notice (JsonNode *node, GError **error)
     return retlist;
 }
 
+static void
+add_affected_snap_to_list (JsonArray *array, guint index, JsonNode *element, void *data)
+{
+    GPtrArray *list = (GPtrArray *) data;
+    const gchar *snap_name = json_array_get_string_element (array, index);
+    g_ptr_array_add (list, g_strdup(snap_name));
```

Comment:
Missing whitespace. Note to self that we really should consider running clang-format over this.